### PR TITLE
Use specific AWS driver for Fog

### PR DESCRIPTION
**Pull request by dalpo**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ group :assets do
 
   # Upload assets to AWS S3
   gem 'asset_sync'
-  gem 'fog', '~>1.20', require: 'fog/aws/storage'
+  gem 'fog-aws', '~> 0.12.0', require: 'fog/aws/storage'
```
Comment (potomak) [inline Gemfile]: What is the difference? I put that line because it was suggested by https://github.com/AssetSync/asset_sync#optimized-fog-loading.

**Comment by potomak** [comment]
Note: at the moment fog is used only to upload compiled assets to S3, but there's an open issue to remove async_assets as suggested by Heroku docs, see https://github.com/potomak/tomatoes/issues/161.

**Comment by dalpo** [comment]
Ok, I didn't notice the #161 issue.

File: Gemfile (hunk 1 above)
Comment (dalpo) [inline Gemfile]: It should relies only on aws as dependency and not all supported drivers by fog.
So bundler hasn't to install every king of fog dependencies...

**Pull request closed without merging**